src/main/java/duke/exception/TodoStringException.java

```diff
@@ -0,0 +1,5 @@
+package duke.exception;
+
+public class TodoStringException extends Exception {
+    //no other code needed
```

Comment:
You could leave out this comment (and similar ones elsewhere) if it is not describing a method.